README.md

```diff
@@ -453,6 +453,21 @@ While most environment variables should be set in the config for a service, some
 govuk-docker run content-publisher-lite env MY_VAR=my_val bundle exec rake my_task
 ```
 
+### How to: install a new release of Ruby
+
+Many of our services use a `.ruby-version` file in conjunction with `rbenv`. When a new version of Ruby is released and we start upgrading our services, you may start seeing the following error when you run commands.
+
+```
+ruby-build: definition not found: x.y.z
+```
+
+Most of our services share a common image, which we needs rebuilding to be aware of the new Ruby version. So, to fix the error, run the following commands, where `<service>` is e.g. `collections-publisher`.
```

Comment:
Is it work stating "docker image"? I know this is a docker repo, but i always find image a very slippary word!

Reply:
Sure. I notice we've used 'Docker container' elsewhere too, so it seems like we have a precedent of being specific. Updated 👍.